tests/js/client/shell/shell-restore-integration.js

```diff
@@ -911,6 +911,62 @@ function restoreIntegrationSuite () {
         db._useDatabase("_system");
       }
     },
+
+    testRestoreEnableRevisionTreesTrue: function () {
+      let path = fs.getTempFile();
+      try {
+        fs.makeDirectory(path);
+        let fn = fs.join(path, cn + ".structure.json");
+
+        fs.write(fn, JSON.stringify({
+          indexes: [],
+          parameters: {
+            name: cn,
+            type: 2
+          }
+        }));
+
+        let args = ['--collection', cn, '--import-data', 'false', '--enable-revision-trees', 'true'];
+        runRestore(path, args, 0); 
+
+        let c = db._collection(cn);
+        let props = c.properties();
+        assertTrue(props.hasOwnProperty("syncByRevision"));
+        assertTrue(props.syncByRevision);
+      } finally {
+        try {
+          fs.removeDirectory(path);
+        } catch (err) {}
+      }
+    },    
+    
+    testRestoreEnableRevisionTreesFalse: function () {
+      let path = fs.getTempFile();
+      try {
+        fs.makeDirectory(path);
+        let fn = fs.join(path, cn + ".structure.json");
+
+        fs.write(fn, JSON.stringify({
+          indexes: [],
+          parameters: {
+            name: cn,
+            type: 2
+          }
+        }));
+
+        let args = ['--collection', cn, '--import-data', 'false', '--enable-revision-trees', 'false'];
```

Comment:
May be add test without --enable-revision-trees to check default value of this parameter?

Reply:
Test added.